.github/packaging/validate_build_output.sh

```diff
@@ -1,6 +1,20 @@
 package_type=${1}
-git clone -b v0.8.23 --depth=1  https://github.com/citusdata/tools.git tools
+
+# Since $HOME is set in GH_Actions as /github/home, pyenv fails to create virtualenvs
+# For this script I set $HOME to /root and then set it back to /github/home
```

Comment:
```suggestion
# Since $HOME is set in GH_Actions as /github/home, pyenv fails to create virtualenvs.
# For this script, we set $HOME to /root and then set it back to /github/home.
```

Reply:
minor